orion.py

```diff
@@ -42,75 +49,32 @@ def orion(**kwargs):
-    logger = logging.getLogger("Orion")
-    logger = orion_funcs.set_logging(level, logger)
-    data = orion_funcs.load_config(kwargs["config"],logger)
-    ES_URL=None
-
-    if "ES_SERVER" in data.keys():
-        ES_URL = data['ES_SERVER']
-    else:
-        if 'ES_SERVER' in os.environ:
-            ES_URL=os.environ.get("ES_SERVER")
-        else:
-            logger.error("ES_SERVER environment variable/config variable not set")
-            sys.exit(1)
-
-    for test in data["tests"]:
-        uuid = kwargs["uuid"]
-        baseline = kwargs["baseline"]
-        match = Matcher(index="perf_scale_ci", level=level, ES_URL=ES_URL)
-        if uuid == "":
-            metadata = orion_funcs.get_metadata(test, logger)
-        else:
-            metadata = orion_funcs.filter_metadata(uuid,match,logger)
-
-        logger.info("The test %s has started", test["name"])
-        if baseline == "":
-            uuids = match.get_uuid_by_metadata(metadata)
-            if len(uuids) == 0:
-                logging.info("No UUID present for given metadata")
-                sys.exit()
-        else:
-            uuids = re.split(' |,',baseline)
-            uuids.append(uuid)
-        if metadata["benchmark.keyword"] == "k8s-netperf" :
-            index = "k8s-netperf"
-            ids = uuids
-        elif metadata["benchmark.keyword"] == "ingress-perf" :
-            index = "ingress-performance"
-            ids = uuids
-        else:
-            index = "ripsaw-kube-burner"
-            if baseline == "":
-                runs = match.match_kube_burner(uuids)
-                ids = match.filter_runs(runs, runs)
-            else:
-                ids = uuids
-
-        metrics = test["metrics"]
-        dataframe_list = orion_funcs.get_metric_data(ids, index, metrics, match, logger)
-
-        merged_df = reduce(
-            lambda left, right: pd.merge(left, right, on="uuid", how="inner"),
-            dataframe_list,
-        )
-
-        csv_name = kwargs["output"].split(".")[0]+"-"+test['name']+".csv"
-        match.save_results(
-            merged_df, csv_file_path=csv_name
-        )
-
-        if kwargs["hunter_analyze"]:
-            orion_funcs.run_hunter_analyze(merged_df,test)
+    level = logging.DEBUG if kwargs['debug'] else logging.INFO
+    logger_instance = SingletonLogger(debug=level).logger
+    logger_instance.info("🏹 Starting Orion in command-line mode")
+    output = run(**kwargs)
+    for test_name, result_table in output.items():
+        print(test_name)
+        print("="*len(test_name))
+        print(result_table)
+
+@cli.command(name="daemon-mode")
+@click.option("--debug", default=False, is_flag=True, help="log level")
+def rundaemon(debug):
+    """
+    Orion runs on daemon mode on port 8000
+    \b
+    """
+    level = logging.DEBUG if debug else logging.INFO
+    logger_instance = SingletonLogger(debug=level).logger
+    logger_instance.info("🏹 Starting Orion in Daemon mode")
```

Comment:
I am not sure if 🏹 is appropriate for orion, may be we want to look for an alternative [here](https://unicode.org/Public/emoji/13.0/emoji-test.txt).

Reply:
Went ahead with bow and arrow as it relates with hunter and orion. Would love to have more suggestions.